### DISCUSSION TITLE
摺疊機的未來趨勢

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS4719-C/Python-Community-GS4719-C#47] [General]
提到貝殼摺疊機，三星無疑是其先驅，除此之外，SONY、小米、OPPO、華為等手機品牌也陸續推出折疊式手機。
市調機構Counterpoint研究智慧型手機市場預測，2023年全球摺疊手機市場將成長10倍，而自2020年就開始推出摺疊機型的三星，將拿下大部分的市場；此外，不僅數量上將大幅增加，就連Apple也可望在2023年推出摺疊手機，若蘋果順利推出，將有助智慧型手機市場進入新的轉折點。
不過摺疊機一直都有價格高昂以及重量較重的缺點QQ
不知道大家有沒有用過?會不會想用呢?(我自己還是很愛我的Iphone11拉哈哈XD

新聞內容: 
[https://www.eprice.com.tw/mobile/talk/4546/5677987/1/](url)
[https://tw.news.yahoo.com/%E6%91%BA%E7%96%8A%E6%89%8B%E6%A9%9F2%E5%B9%B4%E8%A1%9D10%E5%80%8D-%E7%A0%94%E8%AA%BF-%E8%98%8B%E6%9E%9C%E6%91%BA%E7%96%8A%E6%A9%9F2023%E5%8F%AF%E6%9C%9B%E5%95%8F%E5%B8%82-093634207.html](url)

**Comment by pandaiscollin** [comment]
除了重量重跟價格外  對折後的厚度太厚對我來說也是一個問題  因為放口袋後會剩沒多少空間塞其他東西
而且這種手機在摺疊處的精密程度應該也很高(?  如果壞掉的話維修費可能也會比一般手機貴上不少

**Reply by chia-kai-chang** [reply]
我唸研究所時用這個

怎麼一次用兩隻手機 呵

![image](https://user-images.githubusercontent.com/1768891/135754377-a6ca700b-20a9-476e-acac-476c2f4797f7.jpeg)
